src/lightning/pytorch/loggers/neptune.py

```diff
@@ -55,13 +55,13 @@ class NeptuneLogger(Logger):
 
     .. code-block:: bash
 
-        pip install neptune-client
+        pip install neptune
 
     or conda:
 
     .. code-block:: bash
 
-        conda install -c conda-forge neptune-client
+        conda install -c conda-forge neptune
```

Comment:
The conda-forge package has not been renamed yet (I believe this is in progress), so this currently does not work.